plugins/CMakeLists.txt

```diff
@@ -204,14 +204,14 @@ REGISTER_OUT_PLUGIN("out_websocket")
 REGISTER_OUT_PLUGIN("out_cloudwatch_logs")
 REGISTER_OUT_PLUGIN("out_kinesis_firehose")
 REGISTER_OUT_PLUGIN("out_kinesis_streams")
-REGISTER_OUT_PLUGIN("out_prometheus_exporter")
```

Comment:
Intentional change?

Reply:
Could be some merge issue, let me fix it.

Reply:
Still looks wrong in the diff, does this need rebasing?

Reply:
ok, i will rebase it in a whole after Masoud's initial review.